website/docs/reference/dbt-jinja-functions/properties-yml-context.md

```diff
@@ -0,0 +1,34 @@
+---
+title: " About properties.yml context"
+sidebar_label: "properties.yml context"
+id: "dbt-properties-yml-context"
+description: "The context methods and variables available when configuring resources in a properties.yml file."
+---
+
+The following context methods and variables are available when configuring
+resources in a `properties.yml` file.
+
+**Available context methods:**
+- [env_var](/reference/dbt-jinja-functions/env_var)
+- [var](/reference/dbt-jinja-functions/var)
+
+**Available context variables:**
+- [target](/reference/dbt-jinja-functions/target)
+- [builtins](/reference/dbt-jinja-functions/builtins)
+- [dbt_version](/reference/dbt-jinja-functions/dbt_version)
+
+### Example configuration
+
+<File name='properties.yml'>
+
+```yml
+# Configure this model to be materialized as a view
+# in development and a table in production/CI contexts
+
+models:
+- name: dim_customers
```

Comment:
not sure if the yaml is indented so sugg this just in case:

```suggestion
models:
  - name: dim_customers
    config:
      materialized: "{{ 'view' if target.name == 'dev' else 'table' }}
```

Reply:
the rendered format seems like the indentation is a little off for `- name`: https://docs-getdbt-com-git-dbeatty-properties-yml-context-dbt-labs.vercel.app/reference/dbt-jinja-functions/dbt-properties-yml-context#example-configuration

Reply:
<img width="902" alt="Screenshot 2024-07-26 at 10 39 03" src="https://github.com/user-attachments/assets/09584dfa-c91c-44da-a090-cd3b51a320a4">

Reply:
Cool call!

Fixed ✅